chain-signatures/node/src/protocol/mod.rs

```diff
@@ -166,9 +166,22 @@ impl MpcSignProtocol {
         triple_storage: LockTripleNodeStorageBox,
         cfg: Config,
     ) -> (Self, Arc<RwLock<NodeState>>) {
+        let my_address = my_address.into_url().unwrap();
+        let rpc_url = rpc_client.rpc_addr();
+        let signer_account_id: AccountId = signer.clone().account_id;
+        tracing::info!(
+            "initializing protocol with parameters:
+            my_address: {my_address},
+            mpc_contract_id: {mpc_contract_id},
+            account_id: {account_id},
+            rpc_client_url: {rpc_url},
+            signer_account_id: {signer_account_id},
+            cfg: {:?}",
+            cfg
+        );
```

Comment:
Just use the tracing formatter itself otherwise this will probably have weird formatting on the string with all these spaces inside the string:
```suggestion
        tracing::info!(
            my_address,
            mpc_contract_id,
            account_id,
            rpc_url,
            signer_account_id,
            ?cfg,
            "initializing protocol with parameters"
        );
```